Resources/Storage/StorageFactory.py

```diff
@@ -253,13 +260,13 @@ def _getConfigStorageProtocolDetails( self, storageName, protocolSection ):
       configPath = cfgPath( protocolConfigPath, option )
       optionValue = gConfig.getValue( configPath, '' )
       protocolDict[option] = optionValue
-        
+
     # This is a temporary for backward compatibility
-    if "ProtocolName" in protocolDict and not protocolDict['PluginName']:  
+    if "ProtocolName" in protocolDict and not protocolDict['PluginName']:
       protocolDict['PluginName'] = protocolDict['ProtocolName']
-    protocolDict.pop( 'ProtocolName' )  
-        
-    # Evaluate the base path taking into account possible VO specific setting 
+    protocolDict.pop( 'ProtocolName' )
```

Comment:
B.t.w. it would be good if everybody could avoid adding blank spaces here and there... overtime Eclipse is removing them and creates an artificial change which makes reviewing difficult... at least one good point with Eclipse ;-)